packages/arb-token-bridge-ui/src/components/TransferPanel/WithdrawOnlyDialog.tsx

```diff
@@ -0,0 +1,41 @@
+import { useActions, useAppState } from '../../state'
+import { Dialog } from '../common/Dialog'
+import { useWithdrawOnlyDialogStore } from './TransferPanelMain'
+
+export function WithdrawOnlyDialog() {
+  const { app } = useAppState()
+  const { selectedToken } = app
+  const {
+    app: { setSelectedToken }
+  } = useActions()
+  const {
+    isOpen: isOpenWithdrawOnlyDialog,
+    closeDialog: closeWithdrawOnlyDialog
+  } = useWithdrawOnlyDialogStore()
```

Comment:
Mentioned earlier - `useWithdrawOnlyDialogStore` definition should be here instead of TransferPanelMain